bundle/Controller/ContentEditController.php

```diff
@@ -103,7 +103,7 @@ public function createWithoutDraftAction($contentTypeIdentifier, $language, $par
 
         return new ContentCreateView(null, [
             'form' => $form->createView(),
-            'languageCode' => $language,
```

Comment:
It might not be critical, but maybe I'd try to keep some BC here: _add_ `language`, and comment `languageCode` as deprecated ?

Reply:
I'm not sure. 2.0 will change a lot actually, I'd love to make it as clean as possible in 2.0 so that we aren't left with a lot of deprecations till 3.0. I believe it's just a matter of writing a descriptive doc on what is not backwards compatible.